Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -307,8 +307,10 @@ internal static void UpdateLevelKind(MatchExpression matchExpression)
                     DefaultTraces.None;
 
             context.Retry = options.Retry;
+            context.RedactSecrets = options.RedactSecrets;
             context.EnhancedReporting = options.EnhancedReporting;
             context.DynamicValidation = options.DynamicValidation;
+            context.DataToInsert = options.DataToInsert.ToFlags();
```

Comment:
>DataToInsert [](http://example.com/codeflow?start=20&length=12)

Bug! We didn't previously pass this data on the context object. It was apparently unused previously but the recent region populating changes resulted it being consulted in some code paths. Found via unit testing.